apps/industrial-hemp/sections/summary-data-sections.js

```diff
@@ -98,6 +98,34 @@ module.exports = {
       {
         step: '/how-leaves-flowers-destroyed',
         field: 'how-leaves-flowers-destroyed'
+      },
+      {
+        step: '/authorised-witness-details',
+        field: 'authorised-witness-details',
+        parse: (list, req) => {
+          const authorisedWitnessDetails = [
+            req.sessionModel.get('authorised-witness-full-name'),
+            req.sessionModel.get('authorised-witness-uk-telephone'),
+            req.sessionModel.get('authorised-witness-email')
+          ];
+          return authorisedWitnessDetails.filter(element => element).join('\n');
+        }
+      },
+      {
+        step: '/authorised-witness-dbs',
+        field: 'authorised-witness-dbs-information',
+        parse: (list, req) => {
```

Comment:
Because this step is optional and formatDate will throw an exception if the date field is empty, it is necessary to add a check to see if this step would be parsed. 
```suggestion
        parse: (list, req) => {
          if (req.sessionModel.get('authorised-witness-dbs') == 'no') return null 
```